download_and_create_reference_datasets/v02/mito/write_combined_mito_reference_data_hts.py

```diff
@@ -4,8 +4,9 @@
 
 import hail as hl
 
-from download_and_create_reference_datasets.v02.hail_scripts.write_combined_reference_data_ht import join_hts
+from hail_scripts.reference_data.combine import join_hts
 
+VERSION = '2.0.5'
```

Comment:
Should the mito version match the combined?

Reply:
Yeah, we want it to be 2.0.4?

Reply:
The current mito version is`2.0.4`, so I'm gonna leave it at that.